features/step_definitions/all_my_step_definitions_step.rb

```diff
@@ -0,0 +1,25 @@
+Given("the following user exists") do |table|
+    table.hashes.each do |user|
+      User.create(user)
+end
+  end
+  
+  Given("I am on the landing page") do
+    visit '/'
```

Comment:
Please make use of the route helpers provided by Rails in steps like this. In this case:
```
visit root_path
```